docs/protocol/specifications/vm1.md

```diff
@@ -1066,9 +1074,11 @@ Code  | Stack Diagram                  | Cost
 ------|--------------------------------|-----------------------------------------------------
 0xae  | (∅ → hash)                     | 256 + [standard memory cost](#standard-memory-cost)
 
-Computes the [transaction signature hash](data.md#transaction-signature-hash) corresponding to the current input.
+Computes the transaction signature hash corresponding to the current entry. Equals [SHA3-256](blockchain.md#sha3) of the concatenation of the current [entry ID](blockchain.md#entry-id) and [transaction ID](blockchain.md#transaction-id):
```

Comment:
Need to define "current entry." The execution context described above mentions the index of one of the transaction's inputs, but that's outdated; the current entry could also be an interior node, such as  a mux.